modules/rpc/yarpc_options.go

```diff
@@ -52,22 +52,22 @@ func WithOnewayInboundMiddleware(i ...transport.OnewayInboundMiddleware) modules
 	}
 }
 
-func inboundMiddlewaresFromCreateInfo(mci service.ModuleCreateInfo) []transport.UnaryInboundMiddleware {
+func inboundMiddlewaresFromCreateInfo(mci service.ModuleCreateInfo) []middleware.UnaryInbound {
 	items, ok := mci.Items[_interceptorKey]
 	if !ok {
 		return nil
 	}
 
 	// Intentionally panic if programmer adds non-interceptor slice to the data
```

Comment:
can we update this comment to not mention "interceptor" since that language was apparently changed in yarpc?